### PR TITLE
[RAPPS-DB] Add AssaultCube 1.2.0.2

### PR DESCRIPTION
**Pull request by oleg-dubinskiy**
assaultcube

AssaultCube is a free 3D first-person shooter game based on Cube engine.
Works fine in ReactOS, with Mesa3D from Rapps and with VBoxOGL renderer. Although with Mesa3D it has some minor issues: the map at the top right does not render properly (is invisible) and the game has lower FPS than with VBoxOGL.
I guess these are Mesa3D issues, not ReactOS, and on XP/2003 it will show same (or, at least, similar) result, although I did not test it yet.

### CODE REVIEW

**Comment by learn-more** [comment]
does it work without mesa?

**Comment by oleg-dubinskiy** [comment]
Yes, and even better than with Mesa3D. I already said about this.

**Comment by learn-more** [comment]
Then let me rephrase:
Does it work without Mesa and VboxOGL?

**Comment by oleg-dubinskiy** [comment]
> Does it work without Mesa and VboxOGL?

Yes, it does, although with very low FPS:
VirtualBox_ReactOS_17_03_2020_21_14_07

But is it a mandatory requirement to add the games in Rapps? :thinking:

**Comment by learn-more** [comment]
No, but it provides a better experience, as long as we don't have hints implemented that users should install something additionally.
